src/onepasswordconnectsdk/async_client.py

```diff
@@ -0,0 +1,371 @@
+        url = PathBuilder().vaults(vault_id).items(item_id).files().build()
+        response = await self.build_request("GET", url)
+        try:
+            response.raise_for_status()
+        except HTTPError:
+            raise FailedToRetrieveItemException(
+                f"Unable to retrieve item. Received {response.status_code}\
+                             for {url} with message: {response.json().get('message')}"
+            )
+        return self.serializer.deserialize(response.content, "list[File]")
+
+    async def get_file_content(self, file_id: str, item_id: str, vault_id: str, content_path: str = None):
+        url = content_path
+        if content_path is None:
+            url = PathBuilder().vaults(vault_id).items(item_id).files(file_id).content().build()
+        response = await self.build_request("GET", url)
+        try:
+            response.raise_for_status()
+        except HTTPError:
+            raise FailedToRetrieveItemException(
+                f"Unable to retrieve items. Received {response.status_code} \
+                     for {url} with message: {response.json().get('message')}"
+            )
+        return response.content
+
+    async def download_file(self, file_id: str, item_id: str, vault_id: str, path: str):
+        file_object = await self.get_file(file_id, item_id, vault_id)
+        filename = file_object.name
+        content = await self.get_file_content(file_id, item_id, vault_id, file_object.content_path)
+        global_path = os.path.join(path, filename)
+
+        file = open(global_path, "wb")
+        file.write(content)
+        file.close()
+
+    async def get_item(self, item: str, vault: str):
+        """Get a specific item
+
+        Args:
+            item (str): the id or title of the item to be fetched
+            vault (str): the id or name of the vault in which to get the item from
+
+        Raises:
+            FailedToRetrieveItemException: Thrown when a HTTP error is returned
+            from the 1Password Connect API
+
+        Returns:
+            Item object: The found item
+        """
+
+        vault_id = vault
+        if not is_valid_uuid(vault):
+            vault = await self.get_vault_by_title(vault)
+            vault_id = vault.id
+
+        if is_valid_uuid(item):
+            return await self.get_item_by_id(item, vault_id)
+        else:
+            return await self.get_item_by_title(item, vault_id)
+
+    async def get_item_by_id(self, item_id: str, vault_id: str):
+        """Get a specific item by uuid
+
+        Args:
+            item_id (str): The id of the item to be fetched
+            vault_id (str): The id of the vault in which to get the item from
+
+        Raises:
+            FailedToRetrieveItemException: Thrown when a HTTP error is returned
+            from the 1Password Connect API
+
+        Returns:
+            Item object: The found item
+        """
+        url = f"/v1/vaults/{vault_id}/items/{item_id}"
```

Comment:
`[non-blocking]`
It seems that you missed to replace this with the path builder functionality 😜

Reply:
good catch! fixed